### PR TITLE
Getprocess

### PR DESCRIPTION
**Pull request by matusjokay**

### CODE REVIEW

File: security/medusa/l2/evtype_getprocess.c [hunk 1]
```diff
@@ -33,23 +33,98 @@ int process_kobj_validate_task(struct task_struct *ts)
...
+	}
+	get_task_struct(ts_parent);
+	rcu_read_unlock();
+
+	/* recursive call parent's validation if necessary */
+	if (ts == ts_parent)
+		goto init_always_do_direct_getprocess;
+
+	mutex_lock_nested(&(task_security(ts_parent)->validation_in_progress),
+			  task_security(current)->validation_depth_nesting);
```
Comment (Programator2) [inline security/medusa/l2/evtype_getprocess.c:74]: indentation

Comment (Programator2) [reply security/medusa/l2/evtype_getprocess.c:74]: never mind

File: security/medusa/l2/evtype_getprocess.c [hunk 2]
```diff
@@ -33,23 +33,98 @@ int process_kobj_validate_task(struct task_struct *ts)
...
+		put_task_struct(ts_parent);
+		return 1;
+	}
+
+init_always_do_direct_getprocess:
+	put_task_struct(ts_parent);
+
+	get_cmdline(ts, task_security(ts)->cmdline, sizeof(task_security(ts)->cmdline));
 	process_kern2kobj(&proc, ts);
 	retval = MED_DECIDE(getprocess_event, &event, &proc, &proc);
```
Comment (Programator2) [inline security/medusa/l2/evtype_getprocess.c:111]: Missing parent in `MED_DECIDE`

**Review by Programator2 — CHANGES_REQUESTED** [review]
Add parent to `MED_DECIDE`.

**Comment by matusjokay** [comment]
Resolved parent in MED_DECIDE

**Review by Programator2 — APPROVED** [review]

**Pull request merged**